fedml_core/trainer/model_trainer.py

```diff
@@ -8,9 +8,11 @@ class ModelTrainer(ABC):
        2. This class can be used in both server and client side
        3. This class is an operator which does not cache any states inside.
     """
-    def __init__(self, model):
+    def __init__(self, model, args=None):
+        # TODO: Make args mandatory during initialization
         self.model = model
         self.id = 0
+        self.args = args
```

Comment:
Once we set the args as a member variable why do we still need to pass it in the class methods?